src/sentry/static/sentry/app/views/eventsV2/queryList.tsx

```diff
@@ -71,7 +71,10 @@ class QueryList extends React.Component<Props> {
     const {location, organization} = this.props;
     let views = ALL_VIEWS;
     if (organization.features.includes('transaction-events')) {
-      views = [...ALL_VIEWS, ...TRANSACTION_VIEWS];
+      // insert transactions queries at index 2
+      const cloned = [...ALL_VIEWS];
+      cloned.splice(2, 0, ...TRANSACTION_VIEWS);
```

Comment:
Will the result of this be that users with tracing also only see 6 pre built queries?

Reply:
Yea at most 6 pre-built queries.

I can see how it could be awkward to see 5 pre-built queries for users that have no access to APM.

The list I'm working from is at https://app.asana.com/0/1142770666213674/1149458955583705/f

Reply:
@markstory does this impact your pagination work for saved-queries?

Reply:
Not really. I'm planning on using the length of the pre-built views to calculate how many queries I need for the first page.